addons/website_sale/views/crm_team_views.xml

```diff
@@ -16,8 +16,8 @@
                             <div>
                                 <a name="get_abandoned_carts" type="object">
                                     <field name="abandoned_carts_count"/>
-                                    <t t-if="record.abandoned_carts_count.raw_value != 1">Abandoned Carts</t>
-                                    <t t-else="">Abandoned Cart</t>
+                                    <t t-if="record.abandoned_carts_count.raw_value != 1">Abandoned Carts to Recover</t>
```

Comment:
Actually I would remove this t-if t-else. In Odoo we generally use plural to avoid having if / else everywhere. I would therefore always display Abandoned Carts to Recover.